CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 <!-- markdownlint-disable no-duplicate-heading -->
 # Change Log
 
+## 0.7.1
+
+### Bugfixes
+
+* Fixed inline encryption of multiline strings (#337) @jeinwag
+* Prevented throwing an unhandled exception caused by the undefined linter
+  arguments settings
+  ([#142](https://github.com/ansible/ansible-language-server/pull/142)) @ssbarnea
+* Implemented opening standalone Ansible files that have no workspace
+  associated
+  ([#140](https://github.com/ansible/ansible-language-server/pull/140)) @ganeshrn
```

Comment:
```suggestion
  ([ansible/ansible-language-server#142](https://github.com/ansible/ansible-language-server/pull/142)) @ssbarnea
* Implemented opening standalone Ansible files that have no workspace
  associated
  ([ansible/ansible-language-server#140](https://github.com/ansible/ansible-language-server/pull/140)) @ganeshrn
```

Reply:
I am pretty sure that this suggestion will break the lint. Can you please push the fix directly once it passes linting? Editing md in browser is not safe.